routes.py

```diff
@@ -146,17 +146,18 @@ def prepare_escrow_handler(
 @BLUEPRINT.route("/v{}/accept_package".format(VERSION), methods=['POST'])
 @flasgger.swag_from(swagger_specs.ACCEPT_PACKAGE)
 @webserver.validation.call(['escrow_pubkey'], require_auth=True)
-def accept_package_handler(user_pubkey, escrow_pubkey):
+def accept_package_handler(user_pubkey, escrow_pubkey, location=None):
     """
     Accept a package.
     If the package requires collateral, commit it.
     If user is the package's recipient, release all funds from the escrow.
     ---
     :param user_pubkey:
     :param escrow_pubkey:
+    :param location:
     :return:
     """
-    db.update_custodian(escrow_pubkey, user_pubkey)
+    db.add_event(escrow_pubkey, user_pubkey, 'accept_package', location)
```

Comment:
As usual, I have a naming problem :)

This field is a string (VARCHAR), so I don't think we need to use underscores. Spaces are fine.

That said - all our events are package related, and we even have an `escrow_pubkey` foreign key that connects us to a row in the `packages` table. So the word "package" is not required.

Last thing, considering my previous remark about selecting from the events table, I'm thinking on using different event types for courier and recipient. So we can have the following event types:
1. launched
2. couriered
3. received